--- a/config/app.php
+++ b/config/app.php
@@ -91,21 +91,7 @@
     'fallback_locale' => env('APP_FALLBACK_LOCALE', 'en'),
 
     'available_locales' => [
-        'ar' => 'Arabic',
-        'de' => 'German',
-        'en' => 'English',
-        'es' => 'Spanish',
-        'fi' => 'Finnish',
-        'fr' => 'French',
-        'it' => 'Italian',
-        'sv' => 'Swedish',
-        'uk' => 'Ukrainian',
-        'ko' => 'Korean',
-        'lv' => 'Latvian',
-        'nl' => 'Dutch',
-        'no' => 'Norwegian',
-        'pt' => 'Portuguese',
-        'sr' => 'Serbian',
+        'en' => 'English'
     ],
 
     'rtl_locales' => [
